scripts/run_profile_pyuvsim.py

```diff
@@ -66,7 +66,8 @@
     input_uv.ant_1_array, input_uv.ant_2_array = input_uv.baseline_to_antnums(input_uv.baseline_array)
     ants_new = np.unique(input_uv.ant_1_array.tolist() + input_uv.ant_2_array.tolist())
     input_uv.antenna_numbers = ants_new
-    input_uv.antenna_names = ants_new.astype(str)  # Antnames/numbers are going to be messed up by the baseline selection. Unimportant.
+    input_uv.antenna_names = ants_new.astype(
+        str)  # Antnames/numbers are going to be messed up by the baseline selection. Unimportant.
```

Comment:
maybe it would be better here to just move the comment to a separate line above/below the execution so the code doesn't need to be wrapped?

Reply:
this script was removed as per @aelanman 's advice